app/controllers/availabilities/places_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Availabilities
+  class PlacesController < ApplicationController
+    def index
+      @places = Place.all
+    end
+
+    def edit
+      @users = User.all
+      @place = Place.find(params[:id])
+      load_availabilities
+      @time_ranges = TimeRange.order(:start_time)
+      @time_ranges_map = load_timeranges_map
+      @timeslots = load_timeslots
+      @start_time = start_time
+      @end_time = end_time
+    end
+
+    def update
+      Timeslot.where(place_id: params[:id]).each do |timeslot|
+        update_timeslot(timeslot)
+      end
+      redirect_to edit_availabilities_place_path(id: params[:id])
+    end
+
+    private
+
+    def start_time
+      @time_ranges.first.start_time.beginning_of_hour
+    end
+
+    def end_time
+      (@time_ranges.last.end_time - 1).beginning_of_hour + 1.hour
+    end
+
+    def load_timeslots
+      Hash[Timeslot.where(place_id: params[:id]).joins(:time_range)
```

Comment:
I think instead of `Hash[[2, 3], [4, 5]]`, `[[2, 3], [4, 5]].to_h` does the same thing and looks prettier

Reply:
@donjar shld we standardise this everywhere then? Cos I think I did `Hash[` in the code that I wrote too. Or is this just an isolated case?

Reply:
Yes I think we should standardize. I also only know `#to_h` like a few days back haha